src/Bicep.Core.Samples/Files/InvalidDisableNextLineDiagnosticsDirective_CRLF/main.diagnostics.bicep

```diff
@@ -5,19 +5,19 @@ param storageAccount1 string = 'testStorageAccount'
 
 #  disable-next-line  no-unused-params
 //@[00:01) [BCP001 (Error)] The following token is not recognized: "#". (CodeDescription: none) |#|
-//@[00:01) [BCP007 (Error)] This declaration type is not recognized. Specify a metadata, parameter, variable, resource, or output declaration. (CodeDescription: none) |#|
+//@[00:01) [BCP007 (Error)] This declaration type is not recognized. Specify a metadata, parameter, variable, resource, output, or assert declaration. (CodeDescription: none) |#|

```

Comment:
IMO we should avoid exposing "assert" in error messages, given that it's going to be behind a feature flag to start out with

Reply:
That makes sense!

Reply:
I changed the error message back to avoid mentioning "assert"